backend/src/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/implocell/solid-stream/src/ticker"
+
+	"github.com/gorilla/websocket"
```

Comment:
Apparently this is no longer maintained :o

Reply:
Nope, but from what I've read it's the most mature one for now, even if it is no longer maintained.

Reply:
Yup, I got the same impression

It's just a bit sad that the most reliable solution for websockets in go is one that isn't maintained anymore :(
Though I suspect one of the alternatives will get more attention now that this one is on ice